tutorial/header-files.md

```diff
@@ -24,6 +25,61 @@ print('number of threads is :', omp_get_num_threads())
 ```
 Pyccel can compile the Python file with the following command: `pyccel openmp.py --language fortran --flags="-fopenmp"`
```

Comment:
Do we need to use fortran for this to work? If not then can we remove `--language fortran`

Reply:
Actually Fortran is the default language, so we definitely do not need to specify the language.

Reply:
does fopenmp works for fortran only or does  it work with all compilers and os  and for Fortran and C ?
because this way the user can change however he wants

Reply:
I think that we should provide the simplest possible example here. Moreover it is definitely confusing to call this example file `openmp.py`. Hence I recommend
```bash
$ pyccel example.py --openmp
```

Reply:
okay